drivers/gpio/gpio_ite_it8xxx2.c

```diff
@@ -30,6 +30,8 @@ struct gpio_ite_cfg {
 	/* gpio port output type register (bit mapping to pin) */
 	uintptr_t reg_gpotr;
 	uint8_t ngpios;
+	/* GPIO group index */
+	uint8_t idx;
```

Comment:
Nit - can you rename idx to index so it matches the devicetree property name?

Reply:
Done.